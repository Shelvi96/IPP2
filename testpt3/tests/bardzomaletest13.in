NEW ID

4426:527>6
67928>1
7;419>:
06>7:49837
25>334060
734>793;733
7;3;046771>4488
20517>82:315
9;58258>345
1989954>:59
50>156;
55202>:5
42852937>553288
59401>965
8485>0735:009
@394;9:9;69
@9
@:
@8::431834558622393715::48;8
@9:758593:96233;165431446262252
@347001233;26259
@360;
@87522;4;502983
@15548;234;60::;9663806100569
@1;6:83
