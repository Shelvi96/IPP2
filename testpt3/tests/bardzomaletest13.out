0
0
0
1145816813786920
1525082179150390520
202
0
0
12604006173185339
0
